spring-integration-jdbc/src/main/java/org/springframework/integration/jdbc/metadata/JdbcMetadataStore.java

```diff
@@ -0,0 +1,228 @@
+/*
+ * Copyright 2002-2017 the original author or authors.
```

Comment:
Not a big deal from your side, just in case you will follow with the rule in the future to simplify our job:
We use only the current year in Copyright for the new classes.